data/human/jobs.txt

```diff
@@ -3042,3 +3042,138 @@ mission "Bounty Hunting (Marauder X)"
 		payment 750000
 		dialog phrase "generic fleet bounty hunting payment dialog"
+
+
+
+mission "Hunted Warning"
+	invisible
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		attributes "rim" "south" "north" "dirt belt" "core" "frontier"
+	to offer
+		"combat rating" > 8103
+		or
+			not "chosen sides"
+			has "main plot completed"
+	on offer
+		log `Got noticed at a spaceport bar as a "famous warrior." Was warned to look out for bounty hunters or pirates hoping to make a name for themselves.`
+		conversation
+			`You walk into a bar in the spaceport looking for something to eat when a man sitting at a table in the corner looks up and takes notice of you. "It's Captain <last>, the famous warrior! Barkeep, give <first> a drink on my tab."`
+			choice
+				`	"Glad to get some recognition. I work hard out here."`
+					goto recognition
+				`	"Famous warrior?"`
+			`	"Of course! Word travels fast in the galaxy, and word on the hyperspace lanes is that you're quite the fighter."`
+			
+			branch recognition
+				"reputation: Republic" >= 0
+				"reputation: Free Worlds" >= 0
+				"reputation: Syndicate" >= 0
+			
+			`	"It doesn't help that you've pissed off the law, either," someone behind you says, not sounding too happy.`
+			label recognition
+			`	More patrons take notice of the conversation and begin chattering amongst themselves. "Quite the force to be reckoned with, I've heard," you overhear one of them mumble.`
+			`	"You might want to watch your back," another says to you. "Plenty of pirates with more brains than sense might be looking to make a name for themselves by taking out such a famous captain. A person like you has to have some enemies, and you never know who might have put a price on your head. A bounty hunter could retire early on the fat purse they'd get from spacing you."`
+			choice
+				`	"I'll be fine. My ship is strong enough to stand against anything."`
+				`	"Maybe I should go into hiding for a while."`
+					goto hide
+			`	"Oh, I'm sure it is. You wouldn't have gotten this notorious without being able to hold your own."`
+				goto end
+			label hide
+			`	"You'd need to hide for more than just a while, but laying low in a small ship might help to stay off the radar of pirates and bounty hunters."`
+			label end
+			`	After a few more minutes of mumbling, the bar returns to like it was before you entered. The food you ordered arrives and you dig in.`
+				decline
+
+mission "Hunted"
+	invisible
+	landing
+	repeat
+	deadline 30
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		attributes "rim" "south" "north" "dirt belt" "core" "frontier"
+	to offer
+		"combat rating" > 8103
+		"armament deterrence" >= 2
+		"ship scale" = "ship: Light Warship" / 4 + "ship: Medium Warship" / 2 + "ship: Heavy Warship"
+		or
+			and
+				not "Hunted: active"
+				"ship scale" < 8
+				random < 2 + "ship scale"
+			and
+				"ship scale" >= 8
+				random < 10
```

Comment:
Given that the "you're well-armed" branch allows multiple "hunted: active" missions at once, should that fact be used to reduce the offer rate as you obtain more of the mission? 

10% is a pretty high on-landing spawn rate for a mission that will be sending a number of fleets after the player, which is either 1) going to make a rich player richer, or 2) swamp some pour soul that is working their way up to a larger fleet.

e.g.
```
and
  "ship scale" >= 8 + "Hunted: active"
  random < 10 - "Hunted: active"
```

The deadline does limit the number of these that will be active, so an unlucky player might have at most 5-6 active at once, so perhaps this isn't a practical concern.

Reply:
I think the offer rate is fine, but I do like the idea of adding the number of active missions to the ship scale so that you need bigger increasingly bigger fleets in order to have an increasing number of mission active at once.